### PR TITLE
Add doc on downsampling and retention

### PR DESCRIPTION
**Pull request by rkuchan**
Originally: https://github.com/influxdb/influxdb.com/pull/286

### CODE REVIEW

**Comment by rkuchan** [comment]
I still need to edit the links and take care of the sidebar for the new site.

File: content/influxdb/v0.9/guides/downsampling_and_retention.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+In general, InfluxDB wasn't built to process deletes. One of the fundamental assumptions in its architecture is that deletes are infrequent and need not be highly performant. However, InfluxDB recognizes the necessity of purging data that have outlived their usefulness - that is the purpose of RPs.
+
+### Working with RPs
+When you create a database, InfluxDB automatically creates an RP called `default` with an infinite duration and a replication factor set to the number of nodes in the cluster. `default` also serves as the `DEFAULT` RP; if you do not supply an explicit RP when you write a point to the database, the data are subject to the `DEFAULT` RP.
+
+You can also create, alter, and delete you own RPs, and you can change the database's `DEFAULT` RP. See [Database management](/influxdb/v0.9/query_language/database_management/#retention-policy-management) for more on RP management.
+
+> **Clarifying** `default` **vs.** `DEFAULT`
+
+> `default`: The RP that InfluxDB automatically generates when you create a new database. It has an infinite duration and a replication factor set to the number of nodes in the cluster.
```
Comment (beckettsean) [inline content/influxdb/v0.9/guides/downsampling_and_retention.md]: The name of the RP that InfluxDB automatically generates when you create a new database. It has an infinite duration and a replication factor set to the number of nodes in the cluster. It is initially the `DEFAULT` RP as well, but that can be altered.

File: content/influxdb/v0.9/guides/downsampling_and_retention.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+The section below offers a very brief introduction to creating CQs. See [Continuous Queries](/influxdb/v0.9/query_language/continuous_queries/) for a detailed discussion on how to create and manage CQs.
+
+## Combining RPs and CQs - a casestudy
+We have real-time data that track the number of food orders to a restaurant via phone and via website at 10 second intervals. In the long run, we're only interested in the average number of orders by phone and by website at 30 minute intervals. In the next steps, we use RPs and CQs to make InfluxDB:
+
+ * automatically delete the raw 10 second level data that are older than two hours
+ * automatically aggregate the 10 second level data to 30 minute level data
+ * keep the 30 minute level data forever
+
+The following steps work with a fictional [database](/influxdb/v0.9/concepts/glossary/#database) called `food_data` and the [measurement](/influxdb/v0.9/concepts/glossary/#measurement) `orders`. `orders` has two [fields](/influxdb/v0.9/concepts/glossary/#field), `phone` and `website`, which store the number of orders that arrive through the relevant median every 10 seconds.
```
Comment (beckettsean) [inline content/influxdb/v0.9/guides/downsampling_and_retention.md]: "relevant media" not "relevant median"

**Comment by beckettsean** [comment]
minor notes, otherwise looks good to go!

**Comment by beckettsean** [comment]
I do wonder if we need more about the RP syntax for writing to and selecting from non-DEFAULT RPs. Do the RP docs cover this in detail? I don't recall.

**Comment by rkuchan** [comment]
Thanks, @beckettsean!

I made the requested edits and added a discussion about querying from and writing to an RP other than the `DEFAULT` to the `Working with RPs` section. I also added a note about the same topic to the `Create the CQ` section.

The Database Management doc doesn't talk about writing to and selecting from non`DEFAULT` RPs. There is a brief discussion in [Data Exploration](https://docs.influxdata.com/influxdb/v0.9/query_language/data_exploration/#the-basic-select-statement) and there's the somewhat related FAQ on [Frequently Encountered Issues](https://docs.influxdata.com/influxdb/v0.9/troubleshooting/frequently_encountered_issues/#missing-data-after-creating-a-new-default-retention-policy).

**Comment by beckettsean** [comment]
+1

**Pull request merged**